### PR TITLE
feat: add special message for bad indentation

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This should help a little when writing lessons that require a user to create part of a valid code construction.

### CODE REVIEW

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -376,6 +376,7 @@
     "tests-completed": "// tests completed",
     "console-output": "// console output",
     "syntax-error": "Your code raised an error before any tests could run. Please fix it and try again.",
+    "indentation-error": "Your code has an indentation error. You may need to add 'pass' on a new line to form a valid block of code.",
```
Comment (Sembauke) [inline client/i18n/locales/english/translations.json:379]:
```suggestion
    "indentation-error": "Your code has an indentation error, meaning the line spacing isn't consistent. Ensure all lines in a code block are indented the same way.",
```

Comment (Dario-DC) [reply client/i18n/locales/english/translations.json:379]: The message is thought mainly for the case in which camper need to add `pass`.

**Review by Dario-DC — COMMENTED** [review]
I think something is not quite right.. I see `learn.indentation-error` instead of the message `Your code has an indentation error. You may need to add 'pass' on a new line to form a valid block of code.`

**Comment by naomi-lgbt** [comment]
> I think something is not quite right.. I see learn.indentation-error instead of the message Your code has an indentation error. You may need to add 'pass' on a new line to form a valid block of code.

@Dario-DC did you run a full `pnpm run clean-and-develop`?

**Comment by Dario-DC** [comment]
> @Dario-DC did you run a full `pnpm run clean-and-develop`?

Now that I have it works properly, thanks @naomi-lgbt

File: client/i18n/locales/english/translations.json (hunk 1 above)
Comment (Sembauke) [reply client/i18n/locales/english/translations.json:379]: This seems like a very specific use case to me? Or am I wrong @ojeytonwilliams ?

Comment (ojeytonwilliams) [reply client/i18n/locales/english/translations.json:379]: It is, yes. The problem is that some python lessons will have instructions like "create a function `foo`", but if learners just write

```py
def foo():
```

that's not valid python. In this case the hint will appear and, hopefully, they will write

```py
def foo():
  pass
```

which is valid python.

Essentially it's an alternative strategy to writing "please remember to include `pass` in your function stub", or similar, in every lesson like this. We may find that it's not an effective strategy, but I think it's worth trying for now.

**Comment by zairahira** [comment]
Working as intended 👍️

image

**Review by Sembauke — APPROVED** [review]

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Pull request merged**